internal/controller/streams.go

```diff
@@ -203,15 +206,48 @@ func (sm *streamManager) StreamCiliumNetworkFlows(ctx context.Context, ciliumNam
 			err = ciliumFlowCollector.exportCiliumFlows(ctx, *sm)
 			if err != nil {
 				sm.logger.Warnw("Failed to collect and export flows from Cilium Hubble Relay", "error", err)
-				sm.streamClient.disableNetworkFlows = true
+				sm.streamClient.disableNetworkFlowsCilium = true
 				return err
 			}
 		}
 	}
 	return nil
 }
 
-// connectAndStreamCiliumNetworkFlows creates ciliumNetworkFlows client and begins the streaming of network flows.
+// StreamFalcoNetworkFlows handles the falco network flow stream.
+func (sm *streamManager) StreamFalcoNetworkFlows(ctx context.Context) error {
+	var falcoFlow string
+	for {
+		falcoFlow = <-sm.streamClient.falcoEventChan
```

Comment:
This variable doesn't need to be defined out of the loop as it's never accessed outside of the loop, and doesn't need to be preserved across iterations.
```suggestion
	for {
		falcoFlow := <-sm.streamClient.falcoEventChan
```